Jenkinsfile

```diff
@@ -31,6 +31,13 @@ pipeline {
       }
     }
 
+    stage('Scan Go Packages') {
+      steps {
+        sh "./bin/build_security_scan -s High -c 'Medium' -b ${env.BRANCH_NAME}"
```

Comment:
Looking at the other bin/ files, it looks like we use the 'check_' format for file names. So for golint, it's check_style. How about for this we use 'check_security'?

Reply:
is it your intention that the file becomes a general script to run security checks, so that when we have another we'd like to run we also do it from this file?

I'm still not sure I like `check_security` - security means a lot of different things to different people, so if we can find a way to be more clear I think that'd be good